app/models/category_localization.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class CategoryLocalization < ActiveRecord::Base
+  belongs_to :category
```

Comment:
Do we want to add the inverse `has_many` relation to `Category`? Specifically with a dependent strategy like `delete_all` to prevent orphaned translations?

I wonder if in current implementation deleting a category with translations will fail, due to the foreign key constraint.